### PR TITLE
Filip/in place upgrade process improvements

### PR DESCRIPTION
**Pull request by filiphaftek**
# In-place upgrade process for v1.0 - simplified

Based on @michaellzc [PR](https://github.com/sourcegraph/deploy-sourcegraph-managed/pull/587).
I went through it on dev and described the steps which worked.

### CODE REVIEW

File: content/departments/product-engineering/engineering/cloud/devops/managed/upgrade_process.md [hunk 1]
```diff
@@ -531,38 +531,37 @@ git add . && git commit -m "$CUSTOMER: remove $OLD_DEPLOYMENT deployment"
...
 
-Validate all variables are set:
-
-> Are you upgrading to a release candidate build? It's expected to see some errors from this script
-> Use your best judgement to decide whether you're missing something or not
+Then set up a branch for your changes:
 
 ```sh
-./util/validate-env.ts
+git checkout -b $CUSTOMER/upgrade-v<MIJOR.MINOR.PATCH>
```
Comment (sanderginn) [inline content/departments/product-engineering/engineering/cloud/devops/managed/upgrade_process.md]:
```suggestion
git checkout -b $CUSTOMER/upgrade-v<MAJOR.MINOR.PATCH>

File: content/departments/product-engineering/engineering/cloud/devops/managed/upgrade_process.md [hunk 2]
```diff
@@ -531,38 +531,37 @@ git add . && git commit -m "$CUSTOMER: remove $OLD_DEPLOYMENT deployment"
...
 5. [Open a pull request](#13-open-a-pull-request-to-commit-your-changes)
 
-## In-place updates
+## In-place updates (recommended)
 
 This is the upgrade process for running in-place updates. This approach is riskier than the above processes, because rolling back changes takes significantly longer.
 
 Only use this approach for low-risk patch upgrades or docker-compose container resource changes. **Do not use if the patch includes a database change.**
 
 ### 0) Upgrade setup
```
Comment (sanderginn) [inline content/departments/product-engineering/engineering/cloud/devops/managed/upgrade_process.md:542]:
```suggestion
First, ensure that the target version docker-compose manifest is in the `golden` directory. If not, [follow these steps to add it first](v1.1/mi1-1_upgrade_process/#ensure-new-version-of-docker-composeyaml-file-is-in-the-golden-directory).
```

**Review by sanderginn — APPROVED** [review]

File: content/departments/product-engineering/engineering/cloud/devops/managed/upgrade_process.md [hunk 3]
```diff
@@ -531,38 +531,37 @@ git add . && git commit -m "$CUSTOMER: remove $OLD_DEPLOYMENT deployment"
...
 
-**Follow the [general setup](#general-setup) guide.** Then redefine the NEW_DEPLOYMENT variable to match the existing deployment:
-
 ```sh
+export CUSTOMER=<CUSTOMER_FOLDER_NAME>
+export PROJECT_PREFIX=sourcegraph-managed
+export TF_VAR_opsgenie_webhook=<FROM_1PASSWORD>
+export OLD_DEPLOYMENT=$(gcloud compute instances list --project "$PROJECT_PREFIX-$CUSTOMER" | grep -v "executors" | awk 'NR>1 { if ($1 ~ "-red-") print "red"; else print "black"; }')
 export NEW_DEPLOYMENT=$OLD_DEPLOYMENT
 ```
```
Comment (michaellzc) [inline content/departments/product-engineering/engineering/cloud/devops/managed/upgrade_process.md]: You can replace this with the one liner below

```sh
eval $(mg --customer $CUSTOMER workon)
```

🪄

**Review by michaellzc — APPROVED** [review]

File: content/departments/product-engineering/engineering/cloud/devops/managed/upgrade_process.md (hunk 3 above)
Comment (filiphaftek) [reply content/departments/product-engineering/engineering/cloud/devops/managed/upgrade_process.md]: Awesome!

**Pull request merged**